wandb/sdk/wandb_init.py

```diff
@@ -877,11 +874,7 @@ def _attach(
 
     settings: Settings = copy.copy(_wl._settings)
     settings.update(
-        {
-            "run_id": attach_id,
-            "_start_time": response["_start_time"],
-            "_start_datetime": response["_start_datetime"],
-        },
+        {"run_id": attach_id, **response},
```

Comment:
Do we not need the _start_time and _start_datetime attributes? do you know why they were there?

Reply:
we need these so we can access the same directory for the same runs, otherwise we write to a new one.
I tried to be conservative to only use the absolutely necessary settings, but we could in theory expand these.